OracleWebLogic/samples/12213-webtier-apache/Dockerfile

```diff
@@ -46,16 +52,38 @@ COPY container-scripts/* /u01/oracle/container-scripts/
 
 # Use unzip because the base image does not contain a JDK
 # Note that adding unzip does not noticeably increase the size of the image
-RUN yum install -y unzip httpd libaio mod_ssl.x86_64 && \
+RUN yum install -y sudo unzip httpd libaio mod_ssl.x86_64 && \
+    groupadd sudo && \					 
+    useradd -b /u01 -d /u01/oracle -m -G sudo -s /bin/bash oracle && \
+    touch /etc/sudoers && \
+    sed -i /etc/sudoers -re 's/^%sudo.*/%sudo ALL=(ALL:ALL) NOPASSWD: ALL/g' && \
+    sed -i /etc/sudoers -re 's/^root.*/root ALL=(ALL:ALL) NOPASSWD: ALL/g' && \
+    echo "oracle ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
```

Comment:
Having looked into this more carefully, I believe the original line (below) between line#60 and #61 is still needed.
    sed -i /etc/sudoers -re 's/^#includedir.*/## **Removed the include directive** ##"/g' && \

Reply:
As per your original comment at "https://github.com/oracle/docker-images/pull/1630#discussion_r462495380", we removed this line and validated and it worked fine.
Please let me know if its still need to be added back.

Reply:
Sorry for the confusion. I should have given more explanation in my comments.

First of all, either way works for your purposes in the sample in a simple environment. But there are differences.

— Removing the include directive (add back the origin line in the script) prevents accidental override of your new settings with any thing in the directory specified in that line. But it will also remove any additional settings in the includedir even if those setting do not impact your settings. 
— Keeping the include directive (as you have now in this PR) will allow any additional settings to take effect, but has the risk that your settings could be overridden by things in the includedir.
So I think it is slightly safer to remove the include directive, so I suggested to add that line back.

Reply:
Someone would have to modify the image manually in order to override this, so the risk of this occurring is very low. Keeping the includedir directive also maintains standard configuration (as per other images) so maintaining it is recommended so that the behaviour of this particular image matches other images in terms of how `sudo` is configured.

Reply:
that's fine with me.

Reply:
In fact, now that I'm thinking about this more: instead of editing `/etc/sudoers`, you should just add a file to `/etc/sudoers.d` which will be included automatically by the default `includedir` line.

Reply:
Sure, we will try this option and verify.

Reply:
Please review latest commit with subject "Incorporated both comments from @doxiao and @Djelibeybi"